rkt/api_service.go

```diff
@@ -393,65 +527,12 @@ func getBasicPod(p *pod) *v1alpha.Pod {
 				// See https://github.com/coreos/rkt/pull/2331#issuecomment-203540543
 				//
 				// TODO(yifan): Revisit this when using unified cgroup hierarchy.
-				pod.Cgroup = strings.TrimSuffix(cgroup, "/init.scope")
+				v1pod.Cgroup = strings.TrimSuffix(cgroup, "/init.scope")
```

Comment:
I think you can end up with slightly inaccurate information in the following:

1) Running, cgroup cached. 2) Exits, pod mtime unchanged so cache loaded including the running cgroup, cgroup isn't removed here and the cached one is displayed.

I'm not sure if there are other fields like this or the real impact though.